pages/k8s/upgrading.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Upgrading | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, upgrading, track, version
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: upgrading.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+
+# Upgrading
+
+It is recommended that you keep your **Kubernetes** deployment updated to the
+latest available stable version. You should also update the other applications
+which make up the **Canonical Distribution of Kubernetes<sup>&reg;</sup>.**
+Keeping up to date makes sure you have the latest bug-fixes and security
+patches for smooth operation of your cluster.
+
+There is no set cadence to the releases of **Kubernetes**,  but you can check
```

Comment:
Actually there is. The current upstream cadence is a new minor release every quarter. We release the new version of CDK within a week of the upstream release.

Reply:
fixed